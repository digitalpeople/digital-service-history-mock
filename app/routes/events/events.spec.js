import {
  getEvents,
  getEventsById,
} from './events';

jest.mock('./events-handler', () => ({
  getEventsHandler: 'getEventsHandlerMockFunction',
  getEventsByIdHandler: 'getEventsByIdHandlerMockFunction',
}));

describe('getEvents', () => {
  it('should call app.get with correct path', () => {
    expect.assertions(1);

    const app = {
      get: jest.fn(),
    };

    getEvents(app);

    expect(app.get.mock.calls[0]).toMatchSnapshot();
  });
});

describe('getEventsById', () => {
  it('should call app.get with correct path', () => {
    expect.assertions(1);

    const app = {
      get: jest.fn(),
    };

    getEventsById(app);

    expect(app.get.mock.calls[0]).toMatchSnapshot();
  });
});
