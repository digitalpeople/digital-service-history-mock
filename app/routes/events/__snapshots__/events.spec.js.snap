// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getEvents should app.get with correct path 1`] = `
Array [
  "/cars/:id/events",
  "getEventsHandlerMockFunction",
]
`;

exports[`getEventsById should app.get with correct path 1`] = `
Array [
  "/cars/:id/events/:eventId",
  "getEventsByIdHandlerMockFunction",
]
`;
